driver_detection/parse_driver.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import re
+import os
+from datetime import datetime, timedelta
+import time
+import subprocess
+
+from bs4 import BeautifulSoup
+
+
+def download_driver(driver_info, args):
+    try:
+        print('Downloading driver: {}'.format(driver_info['revision_number']))
+        bat_script_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'amd_request.bat')
+        subprocess.run([bat_script_path, driver_info['url'], args.output], timeout=300)
```

Comment:
Лучше взять с небольшим запасом. Драйвер весит почти 0.5Гб
```suggestion
        subprocess.run([bat_script_path, driver_info['url'], args.output], timeout=600)
```